app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ProfileController < ApplicationController
+
+  skip_before_action :verify_authenticity_token
+
+  def show
+      id = params[:id]
+      @profile = User.find(id)
+      @id = id
+      session[:id] = id
+  end
+
+  def edit
+    @profile = User.find(session[:id])
+    @id = session[:id]
+  end
+
+  def update
```

Comment:
We should strongly consider using strong params when we're updating the profile, [documentation here](https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters)